### PR TITLE
CMC Jurisdiction to Sidam User Mapping (AAT,SPROD)

### PR DESCRIPTION
**Pull request by vakepati**
Change description
Main change is added CMC Jurisdiction to AAT and SPROD terra form files  for mapping with SIDAM users.

Does this PR introduce a breaking change? (check one with "x")

[ ] Yes
[x] No

### CODE REVIEW

File: infrastructure/aat.tfvars [hunk 1]
```diff
@@ -1,7 +1,7 @@
 capacity = "2"
 idam_api_url = "https://idam-api.aat.platform.hmcts.net"
 idam_client_redirect_uri = "https://rpe-bulk-scan-processor-sandbox.service.core-compute-sandbox.internal/oauth2/callback"
-supported_jurisdictions = ["SSCS", "BULKSCAN", "DIVORCE", "PROBATE"]
+supported_jurisdictions = ["SSCS", "BULKSCAN", "DIVORCE", "PROBATE","CMC"]
```
Comment (nitinprabhu) [inline infrastructure/aat.tfvars]: space issue here

File: infrastructure/sprod.tfvars [hunk 2]
```diff
@@ -1,4 +1,4 @@
 capacity = "2"
 idam_api_url = "http://idam-api-idam-sprod.service.core-compute-idam-sprod.internal"
 idam_client_redirect_uri = "https://bulk-scan-orchestrator-sprod.service.core-compute-sprod.internal/oauth2/callback"
-supported_jurisdictions = ["SSCS", "BULKSCAN", "DIVORCE", "PROBATE"]
+supported_jurisdictions = ["SSCS", "BULKSCAN", "DIVORCE", "PROBATE","CMC"]
```
Comment (nitinprabhu) [inline infrastructure/sprod.tfvars]: same here as above

File: infrastructure/variables.tf [hunk 3]
```diff
@@ -56,7 +56,7 @@ variable "s2s_name" {
 variable "supported_jurisdictions" {
   type = "list"
   description = "Jurisdictions to be supported by Bulk Scan in the given environment. Bulk Scan will only be able to map these ones to IDAM user credentials"
-  default = ["SSCS", "BULKSCAN", "PROBATE", "DIVORCE", "FINREM"]
+  default = ["SSCS", "BULKSCAN", "PROBATE", "DIVORCE", "FINREM","CMC"]
```
Comment (nitinprabhu) [inline infrastructure/variables.tf]: If you add it here it will enable on prod

Comment (vakepati) [reply infrastructure/variables.tf]: oh :(, thank you ,will remove it now.

**Pull request merged**